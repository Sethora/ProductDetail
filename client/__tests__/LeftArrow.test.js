import LeftArrow from '../src/components/carousel/left-arrow';

describe('Left Arrow', () => {
  test('render the left arrow', () => {
    const wrapper = shallow(<LeftArrow
      slide={() => { }}
      height={28}
      width={28}
    />);
    expect(wrapper).toMatchSnapshot();
  });
});