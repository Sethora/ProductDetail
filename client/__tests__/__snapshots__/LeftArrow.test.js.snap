// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Left Arrow render the left arrow 1`] = `
<styled.div
  height={28}
  id="right"
  onClick={[Function]}
  width={28}
>
  <img
    className="icons"
    src={null}
  />
</styled.div>
`;
